assets/src/edit-story/components/panels/shared/opacityControl.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+/**
+ * WordPress dependencies
+ */
+import { __, _x } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Numeric } from '../../form';
+
+const BoxedNumeric = styled(Numeric)`
+  padding: 6px 6px;
+  border-radius: 4px;
+`;
+
+function OpacityControl({ properties, state, setState }) {
```

Comment:
I think the latest from @samitron7 is that this will get moved into a standalone panel. So maybe we'll need to merge it into a single panel component. But if we to keep it as a separate "Control" - IMHO it should either managed its own state and properties update (props are `selectedElements` and `onSetProperties`) or it should be a controlled component (props are `value` and `onChange`).

Reply:
Yes, it looks like the opacity panel is now a standalone panel, will change this.